### DISCUSSION TITLE
Navigation height

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7381] [Help]
can we change the height of Navigation

**Comment by R2bEEaton** [comment]
Hi @DevrayePatil! Why do you need to change the height of the navigation bar? Is it too big? Too small?

Sort of. You can use the following menu to disable the logo and title which cuts down on some of the height.
![image](https://user-images.githubusercontent.com/34921506/185900939-bb36b01d-9eae-4447-a23a-dce53218a3e5.png)

To shorten it (or extend it) further, you currently would need to use an Embed component on each page including the following code.
```html
<style>
.nav-header { display: none !important; }
.nav { padding: 5px 24px 5px 32px !important; }
</style>
```
The above code would shorten the nav bar quite a lot, while the below code would extend it.
```html
<style>
.nav { padding: 50px 24px 50px 32px !important; }
</style>
```

You can play with this using your browser's "Inspect Element" feature to look at the styles of existing HTML elements. Then, change the style with this Embed component until it looks the way you want. It's not an ideal solution, but if you have a specific use case on only a few pages, this could work fine.
